### PR TITLE
Parser

### PR DESCRIPTION
**Pull request by richtja**
Parser for new data format which combines parameters and constraints in one [file](https://github.com/avocado-framework/cit-ng/blob/master/data_file_example.txt).

### CODE REVIEW

File: varianter_cit/Parser.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+from .Parameter import Parameter
+from .Parameter import Pair
+
+import re
+
+
+class Parser:
+
+    @staticmethod
+    def pars_file(file_object):
```
Comment (clebergnu) [inline varianter_cit/Parser.py]: Any reason for not calling this "parse_file" or even "parse" ?

**Review by clebergnu — COMMENTED** [review]
Hi @richtja, I've a comment about the `pars_file` method name.

File: varianter_cit/Parser.py (hunk 1 above)
Comment (richtja) [reply varianter_cit/Parser.py]: You are right, there is no reason. It could be just calling "parse".

**Comment by clebergnu** [comment]
@richtja there are a few things to fix here:

1) Try to merge the changes currently part of the "Name change of pars_file method" into the "Parser for new input file" commit.  You can do this in a number of ways.  The easiest is to interactively rebase this branch on top of master:

```git rebase -i master```

And move the commit `Name change of pars_file method` right bellow `Parser for new input file`, and marking the action as a "squash".

2) Remove the last commit ("Merge branch 'master' into reader") or merge it into another commit (depending on your intention).

Then, send the resulting work as another PR, marking it as a v2 and stating the changes from v1 is a good practice.

**Pull request closed without merging**